packages/common/src/events/_internal/PropertyMapper.ts

```diff
@@ -167,7 +178,7 @@ export class EventPropertyMapper extends PropertyMapper<
         {
           details: clonedEntity.onlineDetails,
           capacity:
-            clonedEntity.onlineCapacityType === HubEventOnlineCapacityType.Fixed
+            clonedEntity.onlineCapacityType === HubEventCapacityType.Fixed
```

Comment:
I think we need an additional if-block + ternary here to conditionally set the `inPersonCapacity` to `null` when the `inPersonCapacityType` is `fixed` else the provided value of `inPersonCapacity` for in-person & hybrid events to be consistent with what we do for online capacity, else the user can select `unlimited` but any previously saved capacity is still persisted and the form renders the radio group with `fixed` selected upon refreshing.

```ts
    if (
      [HubEventAttendanceType.InPerson, HubEventAttendanceType.Both].includes(
        clonedEntity.attendanceType
      )
    ) {
      obj.inPersonCapacity = clonedEntity.inPersonCapacityType === HubEventCapacityType.Fixed
        ? clonedEntity.inPersonCapacity
        : null;
    }
```